docs/specs/moniker.yml

```
@@ -993,7 +993,7 @@ outputs:
               "name": "Text",
               "monikers": [ "netcore-1.0", "netcore-1.1" ],
               "items": [
-                { "name": "A", "href": "a", "monikers": [ "netcore-1.0", "netcore-1.1" ] }
+                { "name": "A", "href": "a" }
```

Use `undefined` to assert the key does not exist. JSON diff by default does not check additional properties
```suggestion
                { "name": "A", "href": "a", "monikers": undefined }
```